internal/app/machined/main.go

```diff
@@ -68,11 +69,11 @@ func run() (err error) {
 			userdatatask.NewExtraFilesTask(),
 		),
 		phase.NewPhase(
-			"installation",
-			install.NewInstallTask(),
+			"platform tasks",
```

Comment:
nit, probably for a followup pr. but it'd be nice to move these phases names to constants or something a bit more well defined.

Reply:
Yeah I think this might even change TBH. Sort of a quick and dirty hack to have _some_ way to identify a phase for now. Still thinking through the phase/task bits.